packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -187,21 +185,17 @@ export class DocumentStorageService implements IDocumentStorageService {
         handleType: SummaryType,
         path: string[],
         /** Previous snapshot, subtree relative to this path part */
-        previousSnapshot: ISnapshotTree,
+        previousSnapshot: ISnapshotTreeEx,
     ): string {
         assert(path.length > 0, "Expected at least 1 path part");
         const key = path[0];
         if (path.length === 1) {
             switch (handleType) {
                 case SummaryType.Blob: {
-                    const tryId = previousSnapshot.blobs[key];
-                    assert(!!tryId, "Parent summary does not have blob handle for specified path.");
-                    return tryId;
+                    return previousSnapshot.blobs[key];
                 }
                 case SummaryType.Tree: {
-                    const tryId = previousSnapshot.trees[key]?.id;
-                    assert(!!tryId, "Parent summary does not have tree handle for specified path.");
```

Comment:
Why remove these assertions? It now violates the strict return typing of `string`.

Reply:
Also this is sort of an important assertion. We hit this assertion if we are trying to find a tree node using the path that doesn't exist in the prior tree. I'd rather fail here (ASAP) than leter.

Reply:
You are right, I made id non-optional, so did not see ? in there, plus change for blobs is not about Id.